owasp-zap/README.md

```diff
@@ -0,0 +1,38 @@
+# OWASP Zed Attack Proxy
+
+[ZAP](https://www.zaproxy.org/) is a security pen testing tool, which this project uses for active scanning for vulnerabilities. To manually run a scan similar to the ones integrated into [our GitHub Actions](../.github/workflows), run `./manual-scan.sh` from a command line. For full details on running the scan locally, see [Manual run ](#manual-run). Below is a brief explanation of the pieces of this integration and what they are for.
```

Comment:
@jlkravitz I might need some more details since I'm not able to reproduce it. Are there any values in the state drop-down for the user you're logged in as? Do you still get the error if your local `.env` is set to `VUE_APP_MOCK_API=true`?

Either way, I don't think this is a production blocker since it's just limited to the local environment.

Reply:
Yeah let's table. I'll DM you next time it happens to troubleshoot. Thank you!